relay-single.ts

```diff
@@ -541,6 +545,43 @@ export class SingleRelayConnection implements Subscriber, SubscriptionCloser, Ev
         })();
         return chan;
     };
+
+    getSpaceMembers = () => {
+        return getSpaceMembers(this.url);
+    };
+
+    getSpaceMembersStream = () => {
+        const chan = csp.chan<Error | SpaceMember[]>();
+        (async () => {
+            for (;;) {
+                const members = await this.getSpaceMembers();
+                const err = await chan.put(members);
```

Comment:
additional optimization could be that if `members` doesn't change from last time, don't put